auslib/db.py

```diff
@@ -341,13 +382,17 @@ def _prepareInsert(self, trans, changed_by, **columns):
         if self.onInsert:
             self.onInsert(self, "INSERT", changed_by, query, trans)
 
+        potential_signoffs = self.getPotentialSignoffs(None, columns, transaction=trans)
```

Comment:
Yeah, that's not a bad idea. It also makes me wonder if it would be better to have it take a list of rows instead, since we just loop over them in getPotentialSignoffs implementations...I'm not sure there's a use case for treating old and new differently.